communities/usc-project-athena/community-impact-proposal.md

```diff
@@ -0,0 +1,53 @@
+# Summary of the Community's Needs
+  * Community demographics:
+    * Females at USC or in the Southern California area who are interested in coding and tech
+    * At our all female hackathon in April 2017, we hosted over 310 hackers!
+    * Most attendees were from colleges such as USC, UCSD, UCI and UCLA
+    * We had 40 high schoolers attend as well.
+  * Communication:
+    * The community stays in touch through the Facebook group and the slack.
+    * Email will be used for communication between the community and the organizing team.
+  * Culture:
+    * The community is heavily dominated by CS students, but there are also students who are interested in coding, or have a CS minor.
+  
+# Proposal
+  * Timing:
+    * We'd like to host another hackathon next spring, hopefully in February. 
+    * We would also like to host talks, workshops, and a women's conference in the fall semester.
+  * Community demographics:
+    * We're planning on increasing our community size, and we will use social media and campus ambassadors from other schools to help spread the word of our organization and events.
```

Comment:
I'd love to hear more about the campus ambassadors, how you're establishing and leveraging these connections!